docs/guides/pagination/overview.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Pagination - Overview
+---
+
+# Pagination
+
+Whenever we deal with lists in GraphQL, we usually need to limit the number of items returned. Surely, we don't want to send massive lists of
+items that take a considerable toll on the server! The goal of this guide is to help you get going fast with pagination!
+
+## Pagination at a glance
+
+We have always dealt with pagination in different situations. Let us take a look at some of the common ways pagination
```

Comment:
I would hesitate to use "we have always". This could be someone's first shot at pagination and they're coming in with no context.

Reply:
Yeah, that makes sense. I'll probably remove that!